From: Jon Bernard <jbernard@debian.org>
Date: Mon, 6 Dec 2010 21:10:13 -0500
Subject: [PATCH] Fix ppc tracepoint test compilation error
To: ltt-dev@lists.casi.polymtl.ca

On ppc, the tracepoint test fails to compile due to
`register_trace_ust_event_msg' not being defined.

Signed-off-by: Jon Bernard <jbernard@debian.org>
---
 tests/tracepoint/tracepoint_test.h |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/tests/tracepoint/tracepoint_test.h b/tests/tracepoint/tracepoint_test.h
index 6b4dcdb..35f930b 100644
--- a/tests/tracepoint/tracepoint_test.h
+++ b/tests/tracepoint/tracepoint_test.h
@@ -3,6 +3,10 @@
 DECLARE_TRACE(ust_event, TP_PROTO(unsigned int v), TP_ARGS(v));
 DECLARE_TRACE(ust_event2, TP_PROTO(unsigned int v), TP_ARGS(v));
 
+#ifdef __PPC__
+DECLARE_TRACE(ust_event_msg, TP_PROTO(unsigned int v), TP_ARGS(v));
+#endif /* __PPC__ */
+
 struct message {
 	char *payload;
 };
-- 
